resources/content/collections/brands.yaml

```diff
@@ -1,8 +1,14 @@
-title: 'Brand content'
+title: Brands
 propagate: true
+template: rapidez-statamic::brands/show
+layout: layout
 revisions: false
-slugs: false
+route: '/brands/{slug}'
```

Comment:
Is it possible to make this configurable? Should this be de default? Otherwise we should mention this in the readme.

Reply:
I would leave this default to brands, if someone wants it changed they can do so in Statamic.
Updating the readme with the new changes would be a nice anyways!

Reply:
We could also leave it disabled; if someone wants it they can enable it with info in the readme. When you install this package; should brand pages work automatically? Can imagine not everyone want/needs this.

Reply:
True, in that case we could also listen to the brand enabled option in the config when publishing the yaml files?

Reply:
No we couldn't because then we need to either programmaticly edit the yaml file or add a custom route for it. I would say just leave it out for now.

Reply:
And add a description in the readme of how to enable it.

Reply:
👌🏻